README.md

```diff
@@ -2,6 +2,94 @@
 
 # Quad-SDK
 
-**Notice** - This software is currently under construction for an upcoming open source release. Feel free to look around, and stay tuned for more updates!
+## Overview
```

Comment:
Looks great. Do we want to also add the long horizon step climbing figure?

Reply:
This can be a placeholder for now - if I have time I may create a small collage with a few highlights but that's lower priority at the moment.